### PR TITLE
Update teams-ip-phones.md

### PR DESCRIPTION
**Pull request by Leicham**
updated language from suggested correction

### CODE REVIEW

File: Teams/devices/teams-ip-phones.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ Beyond the certification period, Microsoft is committed to make efforts to suppo
 
 The following devices are Certified under the Microsoft Teams phones Certification Program:
 
+> [!NOTE]
+> If your phone is not listed below, manually provisioning the phone may be possible via the device's generic profile and as a SIP device. For more information, see [Configure SIP Gateway](/microsoftteams/sip-gateway-configure).
```
Comment (JohanFreelancer9) [inline Teams/devices/teams-ip-phones.md:43]:
```suggestion
> If your phone is not listed in this table, manually provisioning the phone may be possible via the device's generic profile and as a SIP device. For more information, see [Configure SIP Gateway](/microsoftteams/sip-gateway-configure).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Leicham, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
The product team declined this addition. Closing PR.

**Pull request closed without merging**